modules/addons/grafana/main.tf

```diff
@@ -47,3 +48,9 @@ resource "kubectl_manifest" "tsb_dashboards" {
   for_each   = var.dashboards
   yaml_body  = data.kubectl_path_documents.tsb_dashboards[each.key].documents[0]
 }
+
+resource "local_file" "ssh_tunnel" {
```

Comment:
self-review: now that I think this should be called port-forward instead of ssh-tunnel 🤔